examples/Registration/transformation_matrix.txt

```diff
@@ -0,0 +1,4 @@
+0.81755 0.0660589 -0.0863368 -125.486
```

Comment:
definitely rename to niftyreg_transformation_matrix.txt. It'll use different conventions for other tools